### PR TITLE
Feat/eco 188 added json viewer

### PR DESCRIPTION
**Pull request by Amitkanswal**
Title-SRE checks and UAT changes
Changes-
- updated SRE changes
- updated CSS for devtools
Jira-https://contentstack.atlassian.net/browse/ECO-188

### CODE REVIEW

**Review by kego1992 — APPROVED** [review]

File: src/components/Header.js [hunk 1]
```diff
@@ -35,14 +36,18 @@ const Header = ({ dispatch }) => {
...
+          <div style={{ visibility: "hidden" }}>Dev tools section</div>
+        )}
         <span
           className="devtools"
           data-bs-toggle="modal"
           data-bs-target="#staticBackdrop"
         >
-          <i className="fas fa-tools fa-lg" />
+          <img src={DevtoolsIcon} alt="dev tools icon" title="json preview"/>
+          {/* <i className="fas fa-tools fa-lg" /> */}
```
Comment (abhishek305) [inline src/components/Header.js:50]: if not require remove this comment

File: src/components/devtools.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from "react"
+// import ReactJson from "react-json-view";
+// const ReactJson =typeof window !== "undefined" && require("react-json-view");
```
Comment (abhishek305) [inline src/components/devtools.js:3]: Remove if not required

File: wrap-with-provider.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { Provider } from "react-redux"
+
+import createStore from "./src/store/reducers/state.reducer"
+
+// eslint-disable-next-line react/display-name,react/prop-types
```
Comment (abhishek305) [inline wrap-with-provider.js:6]: remove this too

**Review by abhishek305 — COMMENTED** [review]
cools good

**Review by abhishek305 — APPROVED** [review]

**Pull request merged**